### ISSUE TITLE
Bucket attempts to volunteer dumps of non-factoids

### ISSUE THREAD

**Issue by rcombs** [zigdon/xkcd-Bucket#27]
If someone says the same thing 5 times, even if it's not directed at Bucket and is not a valid factoid trigger, he attempts to volunteer a .txt dump and throws an error. Example:
#xkcd:

```
 <Ariamis> <    rcombs > the mile-away-from-where-you-started club
 <Ariamis> <    rcombs > the mile-away-from-where-you-started club
 <Ariamis> <    rcombs > the mile-away-from-where-you-started club
 <Ariamis> <    rcombs > the mile-away-from-where-you-started club
 <Ariamis> <    rcombs > the mile-away-from-where-you-started club
 <Bucket> That is not a chair listing.
```
#bucketlog:

```
 <Bucket> Volunteering a dump of '< rcombs > the mile-away-from-where-you-started club' for Ariamis in #xkcd
(the spam was accidental on Ariamis's part, for the record)
```

My personal preferred solution to this for Bucket not to send error messages to channel unless his nick was mentioned in the triggering line.

**Comment by dgw** [comment]
Isn't this fixed by #33?

**Comment by zigdon** [comment]
I think it is.
